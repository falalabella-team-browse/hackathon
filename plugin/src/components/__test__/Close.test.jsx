import React from "react";
import { shallow } from "enzyme";
import Close from "../Close";

describe("CloseIcon", () => {
  it("should render properly, with default props ", () => {
    const wrapper = shallow(<Close />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should render properly, with props ", () => {
    const wrapper = shallow(<Close fill="#E2574C" size="20px" />);
    expect(wrapper).toMatchSnapshot();
  });
});
