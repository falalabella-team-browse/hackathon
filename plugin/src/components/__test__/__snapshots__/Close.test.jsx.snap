// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon should render properly, with default props  1`] = `ShallowWrapper {}`;

exports[`CloseIcon should render properly, with props  1`] = `ShallowWrapper {}`;
